src/main/java/com/worldpay/sdk/util/WorldpayLibraryConstants.java

```diff
@@ -70,13 +70,15 @@ private WorldpayLibraryConstants() {
     public static final String LANG_VERSION_PROP =
         "lang.version=" + System.getProperty("java.vm.specification.version");
 
-    public static final String JAVA_VENDOR_PROP = "java.vendor=" + System.getProperty("java.vendor");
-
-    public static final String JVM_VENDOR_PROP = "jvm.vendor=" + System.getProperty("java.vm.vendor");
-
     public static final String API_VERSION_PROP = "api.version=V1";
 
+    public static final String LANG_PROP = "lang=Java";
+
     public static final String OWNER_PROP = "owner=Worldpay";
 
-    public static final String EOF = "\\u001a";
+    public static final String JAVA_VENDOR_PROP = "java.vendor=" + System.getProperty("java.vendor");
+
+    public static final String JVM_VENDOR_PROP = "jvm.vendor=" + System.getProperty("java.vm.vendor");
+
+    public static final String BUILD = "build=";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/nitintiwariwp/worldpay-lib-java/file/1602816680/issues/source?bid=3026869&fileBranchId=3027900#l83)